tsconfig-base.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": [ "es2015" ],
+    "target": "es6",
+    "module": "commonjs",
```

Comment:
This matches what we used before (implicitly by targetting es5). If we do want to start targetting es6 modules that should at least be a separate step and while ES6 in general is well supported, modules are a bit awkward in particular in nodejs so for now I don’t intend to make the switch.